README.md

```diff
@@ -39,71 +38,36 @@ To deploy your test code within the cluster, you must first build and push a con
 Once pushed, override the image within the deployment to deploy your code for testing:
 
 ```console
-$ oc set image -n machine-api-operator deployment/cluster-autoscaler-operator cluster-autoscaler-operator=<YOUR CONTAINER IMAGE>
-deployment.apps/cluster-autoscaler-operator image updated
+oc set image -n machine-api-operator deployment/cluster-autoscaler-operator cluster-autoscaler-operator=<YOUR CONTAINER IMAGE>
 ```
 
 ### Deploy the code to test locally
 
 To deploy your test code locally, you must first disable the existing operator running within the OpenShift cluster:
 
 ```console
-$ oc scale --replicas 0 -n openshift-machine-api deployments/cluster-autoscaler-operator
-deployment.apps/cluster-autoscaler-operator scaled
+oc scale --replicas 0 -n openshift-machine-api deployments/cluster-autoscaler-operator
 ```
 
 Once the operator has been disabled, build your code to test locally and run it on your machine,
 pointing it to the cluster by passing the appropriate `--kubeconfig` flag:
 
 ```console
-$ make build
-  ...
-$ ./bin/cluster-autoscaler-operator --kubeconfig=<PATH/TO/YOUR/CLUSTERS/KUBECONFIG> ...
+make build
+...
+./bin/cluster-autoscaler-operator --kubeconfig=<PATH/TO/YOUR/CLUSTERS/KUBECONFIG> ...
```

Comment:
Not quite sure what to do with this and the two lines above - what does the ... do in the terminal? Is it better to split this into two separate commands?

Reply:
The three dots there are signalling either waiting or expected output, they are not required IMO.
Feel free to remove them

Reply:
I think the first three are in place of the output of the first command, and the second three dots are in place of any possible command line arguments you'd want to pass to the program.

Reply:
we might want to add some description to this example so that readers know this is show a _different_ operator as an example of how to test.

Reply:
Thanks everyone for your comments/reviews, I've implemented your changes now. 
@elmiko I'm not sure what you mean by your last comment - a different operator as opposed to which?

Reply:
my comment is basically speaking to the notion that this part of the instructions (everything that mentions `cluster-autoscaler-operator`) is demonstrating how a user would build a different operator, in this case `cluster-autoscaler-operator`, and then run that operator locally while also running the test suite.

so, imo, we should update this section to describe the scenario that is being given as an example. eg "in this example, we want to run our version of the cluster-autoscaler-operator locally against a remote cluster, and then run the e2e test suite so that we can confirm local changes.". was just a thought while reading all this.